qa/acceptance-tests/src/test/java/io/camunda/it/auditlog/AuditLogIT.java

```diff
@@ -153,149 +164,105 @@ void cleanUp() {
+              assertThat(auditLog.getResult()).isEqualTo(AuditLogResultEnum.SUCCESS);
+            });
   }
 
-  @ParameterizedTest
-  @ValueSource(booleans = {true, false})
-  void shouldFetchAuditLogEntryById(final boolean useRest) {
-    // when
-    // a process instance is created
-    final var processInstanceEvent =
-        startProcessInstance(adminClient, PROCESS_ID, TENANT_A, useRest);
-    final long processInstanceKey = processInstanceEvent.getProcessInstanceKey();
-
-    // then
-    // wait for the audit log entry to be created and available
-    final var piCreated =
-        RecordingExporter.processInstanceCreationRecords()
-            .withIntent(ProcessInstanceCreationIntent.CREATED)
-            .withKey(processInstanceKey)
-            .exists();
-    if (piCreated) {
-      Awaitility.await("Audit log entry is created and can be fetched by ID")
-          .ignoreExceptionsInstanceOf(ProblemException.class)
-          .untilAsserted(
-              () -> {
-                // First, search for the audit log entry to get its ID
-                final var auditLogItems = adminClient.newAuditLogSearchRequest().send().join();
-
-                assertThat(auditLogItems.items().size()).isGreaterThan(0);
-                final var searchedAuditLog = auditLogItems.items().get(0);
-                final String auditLogKey = searchedAuditLog.getAuditLogKey();
-
-                // Fetch the audit log entry by ID
-                final var fetchedAuditLog =
-                    adminClient.newAuditLogGetRequest(auditLogKey).send().join();
-
-                // Verify the fetched audit log entry matches the searched one
-                assertThat(fetchedAuditLog).isNotNull();
-                assertThat(fetchedAuditLog.getAuditLogKey()).isEqualTo(auditLogKey);
-                assertThat(fetchedAuditLog.getEntityType())
-                    .isEqualTo(AuditLogEntityTypeEnum.PROCESS_INSTANCE);
-                assertThat(fetchedAuditLog.getOperationType())
-                    .isEqualTo(AuditLogOperationTypeEnum.CREATE);
-                assertThat(fetchedAuditLog.getCategory())
-                    .isEqualTo(AuditLogCategoryEnum.DEPLOYED_RESOURCES);
-                assertThat(fetchedAuditLog.getProcessDefinitionId())
-                    .isEqualTo(processInstanceEvent.getBpmnProcessId());
-                assertThat(fetchedAuditLog.getProcessDefinitionKey())
-                    .isEqualTo(processInstanceEvent.getProcessDefinitionKey());
-                assertThat(fetchedAuditLog.getProcessInstanceKey()).isEqualTo(processInstanceKey);
-                assertThat(fetchedAuditLog.getTenantId()).isEqualTo(TENANT_A);
-                assertThat(fetchedAuditLog.getResult()).isEqualTo(AuditLogResultEnum.SUCCESS);
-              });
-    }
+  @Test
+  void shouldFetchAuditLogEntryByAuditLogKey() {
+    Awaitility.await("Audit log entry is created and can be fetched by audit log key")
```

Comment:
💭 this is all prep work right? couldn't it even go into the before all?

Reply:
I see your point. Instead of "awaiting" in every test, we can just do it in the `BeforeAll` and then skip this code in the test cases, as we're operating on the assumption that this was already confirmed in the `BeforeAll` method.